igvm/hypervisor_preferences.py

```diff
@@ -66,33 +86,45 @@ def __repr__(self):
 
         return '{}({})'.format(type(self).__name__, args)
 
-    def __call__(self, vm, hv):
+    def get_preference(self, vm, hv) -> Union[float, bool]:
         result = 0
         for other_vm in hv.dataset_obj['vms']:
             if other_vm['hostname'] == vm.dataset_obj['hostname']:
                 continue
+
             if self.values and not all(
                 vm.dataset_obj[a] == v
                 for a, v in zip(self.attributes, self.values)
```

Comment:
Should we rename this to something more speaking like `attr` too ?

Reply:
done